src/Features/Core/Portable/GenerateMember/GenerateConstructor/GenerateConstructorHelpers.cs

```diff
@@ -43,8 +43,7 @@ private static bool IsSymbolAccessible(Compilation compilation, ISymbol symbol)
         {
             if (symbol == null)
                 return false;
-
-            if (symbol is IPropertySymbol { SetMethod: { } setMethod } property &&
+            if (symbol is IPropertySymbol { SetMethod: { } setMethod } &&
```

Comment:
Consider keeping the blank line between `if`s.

Reply:
Ah, this seems like a bug in the IDE0059 code fixer, it seems to have removed it at bunch of places. Let me restore them.